src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.sh

```diff
@@ -15,6 +15,7 @@ __PACKAGES_DIR=${4:-$__TOOLRUNTIME_DIR}
 __TOOLS_DIR=$(cd "$(dirname "$0")"; pwd -P)
 __MICROBUILD_VERSION=0.2.0
 __PORTABLETARGETS_VERSION=0.1.1-dev
+__ROSLYNCOMPILER_VERSION=2.6.0-beta3-62316-02
```

Comment:
This version number is still copied too many places. After your change is this the only one that is needed? Should we figure out a way to set this or allow overriding of this property in the repos? Ideally we would only need to set this in one place for all the scenarios.

Reply:
Unfortunatelly it is not the only place, we still need it in init-tools.cmd as well since we restore the desktop version of the compilers in there as well. That said, we could use the BuildTools version approach and use a txt file to handle this version

Reply:
Now that we can use msbuild we can also create a project file to share between init-tools.sh/cmd and just call build on each and then we could manage the versions in msbuild files.

Reply:
That is a good point, would it be ok if we do that as part of the cleanup of files we won't need anymore like the tool-runtime?

Reply:
Yes I'm fine with doing as part of the clean-up